frontend/src/security/common/AuthContextProvider.tsx

```diff
@@ -0,0 +1,66 @@
+import {ReactElement, useCallback, useEffect, useMemo, useState} from "react";
+import {toast, ToastContainer} from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import {AuthContext, AuthContextProviderValue} from "./AuthContext";
+import axios from "axios";
+import {LoginData} from "../models/LoginData";
+import {Consumer, Run} from "../../components/models/CallbackTypes";
+
+type Props = {
+    children: ReactElement
+}
+
+export default function AuthContextProvider(props: Props) {
+    const {children} = props
```

Comment:
Wenn du destructen willst kannst du das auch direkt in den props machen
```suggestion
export default function AuthContextProvider({children}: Props) {
```